Biasing/python/filters.py

```diff
@@ -39,6 +39,25 @@ def __init__(self,recoil_max_p = 1500.,brem_min_e = 2500.) :
         self.brem_min_energy_threshold = brem_min_e
         self.kill_recoil_track = False
 
+class NonFiducialFilter(simcfg.UserAction):
+    """ Configuration for rejecting events that are fiducial.
+
+    Parameters
+    ----------
+    recoil_max_p : float
+        Maximum momentum the recoil electron can have [MeV]
+    abort_fid_e: bool
+        If turned on, this aborts fiducial events. Else, this tags fiducial events.
```

Comment:
This is not a currently implemented option (although it sounds cool?)

In the interest of speed, I would just remove this comment.
```suggestion
```

Although if you can think of an easy/quick way to implement it, then go ahead.